labs/iot/gps-lab/README.md

```diff
@@ -0,0 +1,53 @@
+# GPS Lab
+
+This folder contains a lab which displays the GPS coordinate in an Azure Map in real time. It collects the GPS coordinates from a GPS sensor attached to a Raspberry Pi and transmits the data
+to IoT hub. The web application containing the azure map reads the data from IoT Hub and displays on the map.
+
+| Author | [Jim Bennett](https://github.com/JimBobBennett) |
+|:---|:---|
+| Target platform   | <ul><li>Raspberry Pi</li><li>Python with Flask</li></ul><i>Optionally you can run this on a PC or Mac using a virtual IoT device</i> |
```

Comment:
I'm not seeing any instructions for using a virtual IoT device - is this supported for this lab?

Reply:
Removed virtual environment. I think this lab should only be used with real device as sending fake GPS data would make this lab almost similar to temp and humidity data virtually